gulpfile.js

```diff
@@ -491,10 +491,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '15.4'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.19.1'))
+    .pipe(replace('[ios.current-app-version]', '2.20.0'))
     .pipe(replace('[android.latest-os-version]', '12'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
-    .pipe(replace('[android.current-app-version]', '2.19.2'))
+    .pipe(replace('[android.current-app-version]', '2.20.0'))
```

Comment:
```suggestion
    .pipe(replace('[android.current-app-version]', '2.20.1'))
```


Will probably be 2.20.1, see latest RC: https://github.com/corona-warn-app/cwa-app-android/releases/tag/v2.20.1-rc.0

Reply:
thank @Ein-Tim